### PR TITLE
Adding the reports page feature without filtering.

### PR DESCRIPTION
**Pull request by WMagnus**
@carolynrehm please review this so merging can occur.

### CODE REVIEW

**Comment by carolynrehm** [comment]
There are merge conflicts from changes in dev that are not reflected in feature-reports. Pull changes and resolve conflicts. Also remove .editorconfig and .code-workspace files and add to .gitignore

File: src/environments/environment.prod.ts [hunk 1]
```diff
@@ -1,5 +1,3 @@
-/** @ignore */
 export const environment = {
    production: true,
-   cms_url: 'http://cms-dev-1861735057.us-east-1.elb.amazonaws.com:9009/CMSforce/as'
```
Comment (carolynrehm) [inline src/environments/environment.prod.ts]: make sure you're not changing environment info

Comment (rankineugeneallen) [reply src/environments/environment.prod.ts]: I'll take care of it.

File: src/app/components/module-index-page/module-index-page.component.ts [hunk 2]
```diff
@@ -66,12 +45,11 @@ export class ModuleIndexPageComponent implements OnInit {
                if (response != null) {
                   this.parseContentResponse(response, module);
                } else {
-                  this.toastr.error('Response was null');
+                  alert('Response was null');
```
Comment (carolynrehm) [inline src/app/components/module-index-page/module-index-page.component.ts]: Were changes is dev pulled? These alerts have been changed and are trying to be reverted

File: src/environments/environment.ts [hunk 3]
```diff
@@ -5,7 +5,7 @@
 /** @ignore */
 export const environment = {
    production: false,
-   cms_url: 'http://localhost:9009/CMSforce/as'
+   cms_url: 'http://localhost:9002'
```
Comment (carolynrehm) [inline src/environments/environment.ts]: Local requests should go through Zuul too

Comment (rankineugeneallen) [reply src/environments/environment.ts]: this should be fixed now.

File: src/app/components/module-index-page/module-index-page.component.ts (hunk 2 above)
Comment (rankineugeneallen) [reply src/app/components/module-index-page/module-index-page.component.ts]: Toastr should be re-implemented now.

File: src/app/components/module-index-page/module-index-page.component.ts [hunk 4]
```diff
@@ -53,33 +66,61 @@ export class ModuleIndexPageComponent implements OnInit {
...
       this.moduleContents.set(module, sortedResponse);
    }
 
+   /**
+    * Description - removes the content from the specified module. It will also send a request to decouple the link between content and module.
+    * @param content - the content being removed
+    * @param module - the module the content is being removed from
+    */
+   removeContentFromModuleIndex() {
+      //looks through the array of links that belongs to content and splices out the module/tag if it finds one.
```
Comment (carolynrehm) [inline src/app/components/module-index-page/module-index-page.component.ts]: remove non documentation comments here

Comment (carolynrehm) [reply src/app/components/module-index-page/module-index-page.component.ts]: and in module-creator-page.component.css, module-index-page.component.css, etc.

Comment (WMagnus) [reply src/app/components/module-index-page/module-index-page.component.ts]: Allen's already fixing it. Please note that many of the things that appear to be changing in components not supposed to be covered by our pull requests that appear to be are in fact a result of odd back-merging from dev into our branch and having to handle odd changes that aren't in fact in dev that we're not sure exactly when or how they occurred.

Comment (WMagnus) [reply src/app/components/module-index-page/module-index-page.component.ts]: It is done.

**Pull request merged**